<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ip-172-31-16-131" failures="4" tests="5" name="Test" time="3.404" errors="0" timestamp="22 Feb 2024 12:11:21 GMT">
  <testcase classname="restAPI.ProjectAPI" name="A_GetAllEmployee" time="1.727"/>
  <testcase classname="restAPI.ProjectAPI" name="B_GetSingleEmployee" time="0.423">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<200>
at restAPI.ProjectAPI.B_GetSingleEmployee(ProjectAPI.java:53)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- B_GetSingleEmployee -->
  <testcase classname="restAPI.ProjectAPI" name="C_CreateEmployee" time="0.552">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<200>
at restAPI.ProjectAPI.C_CreateEmployee(ProjectAPI.java:79)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- C_CreateEmployee -->
  <testcase classname="restAPI.ProjectAPI" name="D_DeleteEmployee" time="0.293">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;200&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<200>
at restAPI.ProjectAPI.D_DeleteEmployee(ProjectAPI.java:112)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- D_DeleteEmployee -->
  <testcase classname="restAPI.ProjectAPI" name="E_TestDeletedEmployee" time="0.378">
    <failure type="java.lang.AssertionError" message="expected:&amp;lt;429&amp;gt; but was:&amp;lt;404&amp;gt;">
      <![CDATA[java.lang.AssertionError: expected:<429> but was:<404>
at restAPI.ProjectAPI.E_TestDeletedEmployee(ProjectAPI.java:136)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- E_TestDeletedEmployee -->
</testsuite> <!-- Test -->
